config/seata-config-core/src/test/java/org/apache/seata/config/FileConfigurationTest.java

```diff
@@ -52,13 +52,20 @@ public void onChangeEvent(ConfigurationChangeEvent event) {
             }
         });
         System.setProperty(dataId, String.valueOf(!value));
-        countDownLatch.await(2, TimeUnit.SECONDS);
-        System.setProperty("file.listener.enabled", "false");
+        countDownLatch.await(10, TimeUnit.SECONDS);
```

Comment:
I think you should add a synchronous assertion of the return value or the first value change.
In the case of past failures, one possible reason is that the configCache update was not triggered after await(2s), causing fileConfig.getBoolean(dataId) to return the cached old value. In the CachedConfigurationChangeListener assertion is not effective, because in the asynchronous link, every change in the value of the trigger, but the timing of the trigger is not controllable. Even if the test process exits, the asynchronous thread may not execute.

Reply:
done